pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "hexlet-code"
+version = "0.1.0"
+description = ""
+authors = ["AlexMomot-717 <momot-717@yandex.ru>"]
+readme = "README.md"
+packages = [
+    {include = "brain_games"},
+]
+
+[tool.poetry.scripts]
+brain-games = "brain_games.scripts.brain_games:main"
+python = "3.10"
```

Comment:
`python = "3.10"` is not a script (it's a dependency) and should be under `[tool.poetry.dependencies]` section  

like  

```toml
[tool.poetry.dependencies]
python = "~3.10"
```

Reply:
[more](https://peps.python.org/pep-0440/#compatible-release) about `~3.10`